it/test/controllers/manageAgent/AreYouSureISpec.scala

```diff
@@ -94,8 +109,8 @@ class AreYouSureISpec extends ISpecBase with HtmlComponentHelpers with ListYears
         document.select(thisAgentSelector).text() shouldBe thisAgentText(listYear = "2017")
       }
 
-      s"has text on the screen of '${theyWillText(otherListYear = "2023")}'" in {
-        document.select(theyWillSelector).text() shouldBe theyWillText(otherListYear = "2023")
+      s"has text on the screen of '${theyWillText(otherListYear = "2017")}'" in {
+        document.select(theyWillSelector).text() shouldBe theyWillText(otherListYear = "2017")
```

Comment:
There's only tests for 2017 and 2023 that I can see. Please can you add tests for when otherListYear=2026 (in English and Welsh)